src/components/wishList/WishListView.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import AlbumGrid from '../AlbumGrid';
+import axiosInstance from '../../apis/axiosClient';
+
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
```

Comment:
According to the lint message from Github Actions we can remove this line now that it's no longer used